API_Guidelines.md

```diff
@@ -81,13 +89,20 @@ Payload
 :
 ```javascript
 {
-  "movies": {
+  "movie": {
     "id": 2,
     "title": "Goodfellas",
     "year": "1990"
   }
 }
 ```
+Payload (If no data is found, then `findRecord` returns an null value)
+:
+```javascript
+{
+  "movie": null
```

Comment:
remember, these are API guidelines, regardless o framework. what's the issue with it being an error?